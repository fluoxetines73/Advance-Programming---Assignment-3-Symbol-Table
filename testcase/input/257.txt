INSERT d number
LOOKUP d
RPRINT
ASSIGN d 123.8.7
LOOKUP d
INSERT f_FF number
INSERT z string