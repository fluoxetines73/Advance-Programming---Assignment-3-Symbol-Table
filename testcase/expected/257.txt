Invalid: ASSIGN d 123.8.7